src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/links/LinkInfo.java

```diff
@@ -69,11 +69,6 @@
      */
     public Content label;
 
-    /**
-     * True if the link should be strong.
-     */
-    public boolean isStrong = false;
-
```

Comment:
I guess this is the counterpart on adding the style into the `LinkInfoImpl`.  Yes, I agree this is the least ugly way to handle the improvement.